### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by Adam-DIY** [thelastoutpostworkshop/gpio_viewer#144] [Q&A]
i. I got following error when trying to compile code for ESP32S NodeMcu:
![image](https://github.com/user-attachments/assets/adad60c9-c250-4a96-9d78-9cc40e51e20d)

Looks like a syntax error (?)
Can You pls help?

**Comment by thelastoutpostworkshop** [comment]
Look at the [tested on section](https://github.com/thelastoutpostworkshop/gpio_viewer#last-tested-to-be-working-on-), make sure you are using the latest versions of the Arduino Core by Espressif and the other libraries.
